### PR TITLE
Austin.Intermediate.Fall.Kendra.Moore

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
today

File: apps/02RockPaperScissors.js [hunk 1]
```diff
@@ -7,10 +7,41 @@ prompt.start();
...
+    } else {
+    return 'Hand two wins!';
+   }
+    }
+
+if (hand1 === 'scissors') {
+if (hand2 === 'paper') {
+  return 'Hand one wins!';
+} else {
+return 'Hand two wins!'
```
Comment (pieroapretto) [inline apps/02RockPaperScissors.js:41]: Nicely done!

Please make sure to format this using tab so you're code is easier to read.

File: homework/01LessonOne.js [hunk 2]
```diff
@@ -105,7 +108,7 @@ var notNotTrue;
 //
 // What is the difference between the "and" and the "or" operators? Explain why you would use each of them.
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: "and" is used to evaluate if both statements are true while "or" is to evaluate if one or both expressions are true
```
Comment (pieroapretto) [inline homework/01LessonOne.js:111]: Correct!

File: homework/01LessonOne.js [hunk 3]
```diff
@@ -143,8 +146,8 @@ var notNotUndefined;
 //
 // Explain truthiness and falsiness in your own words. Provide an example for each.
 //
-// Your Answer Goes Here:
-//
+// Your Answer Goes Here: truthiness is a value that translates to true when evaluated in a Boolean context. All values are truthy unless they are defined as falsy (i.e., except for false, 0, "", null, undefined, and NaN).
+//if(true), if(false)
```
Comment (pieroapretto) [inline homework/01LessonOne.js:150]: Great answer!

damn

File: homework/02LessonTwo.js [hunk 4]
```diff
@@ -68,7 +89,7 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 //
 // What is a parameter? What is an argument? Is there a difference between the two?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: Parameters are the names listed in the function while an arguments is the real value in the function
```
Comment (pieroapretto) [inline homework/02LessonTwo.js:92]: yes!

**Pull request closed without merging**